### PR TITLE
feat(common): remove fdp

### PR DESCRIPTION
**Pull request by McDaddy**
## What type of PR is this?

- [x] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## What this PR does / why we need it:
entirely remove fdp

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

File: shell/app/layout/pages/page-container/components/sidebar.tsx [hunk 1]
```diff
@@ -45,7 +46,7 @@ const AppCenterEl = () => {
     dataCenter: 'duoyun',
     microService: 'weifuwu1',
     orgCenter: 'guanli',
-    diceFdp: 'kuaishuju',
+    [dataEngineerInfo.name]: 'kuaishuju',
```
Comment (daskyrk) [inline shell/app/layout/pages/page-container/components/sidebar.tsx:49]: rename this icon name

File: cli/lib/build.ts [hunk 2]
```diff
@@ -353,18 +382,35 @@ export default async (options: { local?: boolean; image?: string; enableSourceMa
...
 
     if (rebuildList.includes('shell')) {
       localIcon();
     }
+
+    externalModules.forEach(({ nginx, env }) => {
+      if (nginx) {
+        const nginxPath = path.resolve(process.cwd(), 'nginx.conf.template');
+        const nginxContent = fs.readFileSync(nginxPath).toString('utf8').split(EOL);
+        nginxContent.splice(-1, 0, nginx);
```
Comment (daskyrk) [inline cli/lib/build.ts:402]: Should add some comment as insert mark, because nginx route match has order.

File: nginx.conf.template [hunk 3]
```diff
@@ -156,27 +155,6 @@ server {
...
-        proxy_set_header org $1;
-        proxy_pass http://${OPENAPI_ADDR};
-        proxy_http_version 1.1;
-        proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "upgrade";
-        proxy_set_header Host $host;
-    }
-
-    # redirect repo address to gittar when user agent is git
-    location ~ ^/[^/]*/dop/ {
```
Comment (daskyrk) [inline nginx.conf.template:174]: not delete this rule, is useful.

File: cli/lib/build.ts (hunk 2 above)
Comment (McDaddy) [reply cli/lib/build.ts:402]: from my perspective，I think place the fdp proxy in the end of nginx file should have no impact on the route logic.

**Review by daskyrk — APPROVED** [review]

**Pull request merged**